tutorials/cross_cloud_on_prem/README.md

```diff
@@ -0,0 +1,220 @@
+
+## Prerequisites
+This tutorial assumes the reader has a basic familiarity with the Linux command line, Google Cloud Compute Engine, and AWS EC2. 
+
+We also assume the reader has installed PerfKitBenchmarker. If not, we ask the reader first read through the Beginner Walkthrough.
+
+## What You’ll Do
+This lab presents a process for easily performing cross-cloud network performance benchmarking by using PerfKitBenchmarker.
+
+In this example scenario, we are interested in evaluating TCP throughput performance between Google Cloud and AWS in the Virginia(USA) geographical area. Google Cloud and AWS both host cloud regions here, with us-east4 and us-east-1, respectively. We also consider an example scenario of evaluating TCP throughput performance between a local on-premises machine and a Google Cloud VM. 
+
+Finally, we review how we can use the cron utility to regularly perform benchmarks, allowing us to actively monitor a performance metric of interest.
+
+In this lab, we will:
+- Create a set of benchmark configuration files
+- Run benchmarks for AWS <—> Google Cloud TCP throughput
+- Run benchmarks for On-Prem <—> Google Cloud TCP throughput
+- Set up a cron job to schedule PerfKitBenchmarker runs
+
+## What You’ll Need
+For this tutorial, you’ll need:
+- A Google Cloud project
+- An AWS account
+- Installation of PerfKitBenchmarker
+
+## Cross-Cloud Benchmarking Example
+### Verify Google Cloud/AWS CLI and PerfKitBenchmarker Installation
+Before proceeding, verify that the Google Cloud and AWS CLIs are authenticated on the machine which will be running PerfKitBenchmarker. Also, verify that PerfKitBenchmarker is installed and the appropriate Python environment is active, if appropriate.
+### Create Benchmark Configurations
+First, we need to create benchmark configurations for PerfKitBenchmarker to run. In general, we recommend specifying these with YAML files, as they allow for easy sharing and reproduction of testing configurations in the future, if necessary. 
+
+As mentioned, we will be using the iPerf benchmark. The default YAML configuration for all benchmarks in PerfKitBenchmarker are available through their respective Python benchmark file, defined with the BENCHMARK_CONFIG variable. For Linux benchmarks, these files are located in the perfkitbenchmarker/linux_benchmarks directory. Below is the default iPerf configuration.
+
+    iperf:
+      description: Run iperf
+      vm_groups:
+        vm_1:
+          vm_spec: *default_single_core
+        vm_2:
+          vm_spec: *default_single_core
+
+We will be extending this default configuration with several other options. First, create a new directory for storing our configurations. Then, create a new file in this directory titled example.yaml and copy the following text to the file.
+
+    iperf:
```

Comment:
This tutorial provides good descriptions/usage of yaml files, which are a great way to run pkb that our other tutorials don't cover well at all. Could you also add a couple of these yaml files to this pull request / the tutorial folder as examples?